docs/_documentations/remotedeploy-single.md

```diff
@@ -38,13 +36,10 @@ This command requires various flags to specify where and what to install which w
 
 # Pre-requisites
 
-- For OpenShift, Codewind is required to run as privileged and as root because it builds container images. In addition, you must create the namsepace first. 
-    Run the following commands where `<namespace>` is the namespace into which you plan to install Codewind:
-    1. Create the namespace by entering: `oc create namespace <Codewind-namespace>`.
-    2. Run the following commands on only the Codewind namspace: 
-        - Switch to your Codewind-only namespace using: `oc project <Codewind-namespace>`.
-        - To enable privileged containers, enter `oc adm policy add-scc-to-group privileged system:serviceaccounts:<namespace>`.
-        - To enable containers to run as root, enter `oc adm policy add-scc-to-group anyuid system:serviceaccounts:<namespace>`.
+- For OpenShift, Codewind is required to run as privileged and as root because it builds container images. Run the following commands on OpenShift and IBM Cloud only. The `<namespace>` variable is the namespace into which you plan to install Codewind:
```

Comment:
I don't see the 'In addition, you must create the namespace first' (or variant) instruction in this new draft.

Reply:
You do not need to create the namespace first anymore (but you still can, it's optional).

Reply:
You no longer need to create the namespace first.